### DISCUSSION TITLE
postal code not used in components

### DISCUSSION THREAD

**Discussion by stefanofinetti** [spatie/geocoder#98] [Ideas]
For international, non US address, the street name may not be enough to pinpoint the exact location.
In google apis, under components, there is the possibility to specify postcode and locality as component:postcode=xxxxx or component:locality=xxxxx.

Unfortunately, the package does not use any of them besides component:country=xx which is not enough, at least not for Italy addresses.

**Comment by freekmurze** [comment]
Feel free to PR improvements 👍
